app/pkg/build.gradle

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2020 Google LLC, MobilityData IO
```

Comment:
What's the project policy on copyright headers?  I noticed it seems to be hit or miss across the codebase.  I'm wondering if I should go update all my files.  Also, I'm a Google employee but I've also signed the CLA, so I guess the copyright goes to MobilityData, yeah?

Reply:
To my knowledge, past policy has been contributor retains copyright (Google has handled contributions this way in the past) and updates copyright header to include their organization and updates the date range to reflect copyright dates.

So if this contains copy of code from the existing project it would be:

`Copyright 2020-2022 Google LLC, MobilityData IO`

If it's brand new code without any existing project code it would be:

`Copyright 2022 Google LLC`

Reply:
Oh, and CLA contains provisions that allow MobilityData irrevocable re-use of code as part of this project.

Reply:
Addressed in 8f26673749679ad2efd01fc48fad89248c15a794.